codec-http/src/test/java/io/netty/handler/codec/http/websocketx/WebSocket08EncoderDecoderTest.java

```diff
@@ -154,4 +195,16 @@ private void testBinaryWithLen(EmbeddedChannel outChannel, EmbeddedChannel inCha
         }
         binFrame.release();
     }
+
+    private void transfer(EmbeddedChannel outChannel, EmbeddedChannel inChannel) {
+        // Transfer encoded data into decoder
+        // Loop because there might be multiple frames (gathering write)
+        while (true) {
```

Comment:
please use the same style we do in Netty: `for (;;) { ... }`

Reply:
done